client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/usePersonalInfoTab.ts

```diff
@@ -1,106 +1,84 @@
 import Swal from 'sweetalert2';
 import { useSelector } from 'react-redux';
+import axios from 'axios';
 
-import axios from 'Utils/axios';
 import logger from 'logger/logger';
 import { initDBAddress } from 'models/DBAddress';
-import { Service, Severity } from 'models/Logger';
+import { Severity } from 'models/Logger';
 import StoreStateType from 'redux/storeStateType';
 import Occupations from 'models/enums/Occupations';
 import { setInvestigatedPatientId } from 'redux/Investigation/investigationActionCreators';
+import { PersonalInfoDbData } from 'models/Contexts/PersonalInfoContextData';
+import InvestigationMainStatus from 'models/enums/InvestigationMainStatus';
+import { setFormState } from 'redux/Form/formActionCreators';
+import useComplexitySwal from 'commons/InvestigationComplexity/ComplexityUtils/ComplexitySwal';
+import { InvestigationStatus } from 'models/InvestigationStatus';
+import { setIsLoading } from 'redux/IsLoading/isLoadingActionCreators';
 
 import useStyles from './PersonalInfoTabStyles';
-import { usePersoanlInfoTabParameters, usePersonalInfoTabOutcome } from './PersonalInfoTabInterfaces'; 
+import { usePersoanlInfoTabParameters, usePersonalInfoTabOutcome } from './PersonalInfoTabInterfaces';
+import personalInfoValidationSchema from './PersonalInfoValidationSchema';
 
 const usePersonalInfoTab = (parameters: usePersoanlInfoTabParameters): usePersonalInfoTabOutcome => {
 
     const classes = useStyles({});
 
-    const userId = useSelector<StoreStateType, string>(state => state.user.id);
+    const userId = useSelector<StoreStateType, string>(state => state.user.data.id);
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
+    const investigatedPatientId = useSelector<StoreStateType, number>(state => state.investigation.investigatedPatient.investigatedPatientId);
+    const investigationStatus = useSelector<StoreStateType, InvestigationStatus>((state) => state.investigation.investigationStatus);
+
+    const { complexityErrorAlert } = useComplexitySwal();
 
     const { setInsuranceCompanies, setPersonalInfoData, setSubOccupations, setSubOccupationName, setInvestigatedPatientRoles,
-            setCityName, setStreetName, setStreets, occupationsStateContext, setInsuranceCompany,
+        setCityName, setStreetName, setStreets, occupationsStateContext, setInsuranceCompany,
     } = parameters;
 
     const fetchPersonalInfo = (reset: (values?: Record<string, any>, omitResetState?: Record<string, boolean>) => void,
-                               trigger: (payload?: string | string[]) => Promise<boolean>
-                              ) => {
-        logger.info({
-            service: Service.CLIENT,
-            severity: Severity.LOW,
+        trigger: (payload?: string | string[]) => Promise<boolean>
+    ) => {
+        const occupationsLogger = logger.setup({
```

Comment:
I think you should change the logger name

Reply:
I think the name describes the logger pretty accurately. What name do you suggest?